docgen/src/examples/tourism/App.js

```diff
@@ -17,8 +17,8 @@ import React, {PropTypes} from 'react';
 import GoogleMap from 'google-map-react';
 import {fitBounds} from 'google-map-react/utils';
 
-import Rheostat from 'rheostat';
 import {withUrlSync} from '../urlSync';
+import Range from '../../../../stories/3rdPartiesIntegration.stories';
```

Comment:
Would rather have an examples/Slider.js (if we want to "hide" long implementation) that we reuse here and avoid mixing stories with examples. Otherwise we will forget that link between two parts of the code when updating one or the other